newsletter/_posts/2018-04-10-WLCG-HSF-Naples.md

```diff
@@ -0,0 +1,147 @@
+Systems like SWAN have had success in bringing notebook technology to
+HEP ROOT analysis and now plans R&D to scale up to future challenges. In
+the meantime, data science tools like Apache Spark clusters achieve a
+scale and ease of use which is admirable and our community would like
+to emulate. albeit that this is much harder to achieve with HEP
+analysis data. There was a fascinating talk and live demonstration of the Dask
+system for cluster based data analysis and analytics in the Python ecosystem.
+
+Back at the production end of HEP, where data is taken from the detector
+and reconstructed, we heard about the use of FPGAs to perform surprisingly
+complicated reconstruction tasks in the LHCb RICH. Advances in making such
+devices easy to program offer the prospect of more wide-spread use
+of this technology in the future. Looking at other reconstruction challenges
+modern hardware could be used more efficiently if the right algorithms
+are found; that might include replacing some traditional programmed
+solutions with *machine learning* that can usually run well on
+wide vector registers, multiple CPU cores or even GPUs. That challenge might also be
+faced more effectively in common through the use of common toolkits,
+such as VecCore, that help code adapt to the changing hardware landscape.
+An intense discussion about the strategy to be used here, and how much specific HEP solutions
+are really required, was had and typifies many of the open questions
+our R&D programme needs to address. 
+
+<!-- TrickTrack, ACTS mention? -->
+
+For producing simulated events there were two full sessions. 
+Producing events with better underlying physics is needed for both
+HL-LHC and the intensity frontier programs, where accuracy of understanding
+rare processes is a key to success. In addition, there is a need to speed
+up simulation. This takes a few forms: better use of current hardware,
+tested via the GeantV R&D line and the use of approximate techniques that reproduce
+complex processes more cheaply. Machine learning again looks very promising
+here, particularly for calorimiter simulation. Past fast simulation efforts
+have been very detector specific so the challenge is how to make faster
+techniques more generic, including the process of training networks and
+performing validation. There is a very active community here and a vigorous
+and diverse R&D is underway.
+
+The Software Development session reviewed progress made by the
+HSF's [Packaging Working Group](/activities/packaging.html){:target="_naples_packaging_wg"}. A common
+solution here will be of great benefit to our R&D projects in the future
+as it will allow a wider base of users to more easily test early
+releases. A decision was also made to consolidate information and best
+practice on performance monitoring and tooling, which will help developers
+across the field.
+
+Helping establish best practice for development is one part of the training 
```

Comment:
This paragraph has a few trivial typos and I would put plurals in a couple of places: ... best practices ... training challenges

Reply:
Usually "best practice" is singular.